UPGRADING.md

```diff
@@ -6,47 +6,12 @@ While we specify a version of Lemmy, pict-rs, postgres, etc. at the point in tim
 
 While you are not forced into running the specific versions, we do not go through thorough testing on all version compatibility matrices, so please make your best judgement and always backup before performing updates.
 
-### Upgrading to 1.5.6 (Lemmy 0.19.10)
+For each new version run the following steps in your local `lemmy-ansible` folder:
```

Comment:
Lets add a heading above this, like: `## General Upgrade Instructions`

and maybe a little paragraph saying to double check the Specific Version instructions below, and if the version is not listed then it's 100% safe to deploy.

Something I also think we are missing is if there are database migrations in the version or not. As they would be the things an admin would want to lookout for. As these are the "Main" upgrade docs that Lemmy links too.

Reply:
Updated